yoti_python_sdk/endpoint.py

```diff
@@ -6,30 +7,45 @@ class Endpoint(object):
     def __init__(self, sdk_id):
         self.sdk_id = sdk_id
 
-    def get_activity_details_request_path(self, decrypted_request_token):
+    def get_activity_details_request_path(
+        self, decrypted_request_token, no_params=False
+    ):
+        # TODO: Remove no_params in 3.0.0 and get rid of the query parameters in the formatted
```

Comment:
I'm not sure we should be checking in TODO tags into Git (if we can help it)
We should probably use Jira releases to tack this sort of stuff?

Reply:
Yep probably - I'll fix this now